go.mod

```diff
@@ -384,3 +384,7 @@ require (
 	github.com/ziutek/mymysql v1.5.4 // indirect
 	golang.org/x/exp v0.0.0-20230321023759-10a507213a29
 )
+
+replace go.viam.com/api => /Users/patriciastrutz/Development/api
```

Comment:
[minor] you should be able to remove be able to remove both replace lines. The api can be updated to the latest tag and the rdk replace is not necessary

Reply:
Whoops forgot that this would get included in the commit